app/services/editions/prefills/PrefillHelper.scala

```diff
@@ -0,0 +1,67 @@
+package services.editions.prefills
```

Comment:
What's the reason for this change? That is, why are we extracting `geneneratePrefillQuery` out from `Capi.scala`?

Reply:
I wanted to have a package related to prefills solely